deployments/kubernetes/Dockerfile.istio-cni-repair

```diff
@@ -0,0 +1,10 @@
+# hadolint ignore=DL3007
```

Comment:
You shouldn't need yet another image. We're trying to reduce the number of image.
Include this new binary into the image that is already being built.

Reply:
Merged.